docker/Dockerfile

```diff
@@ -106,14 +106,17 @@ RUN \
         boost-iostreams boost boost-date-time \
         boost-test boost-graph boost-signals \
         boost-program-options boost-math \
-        openssh openssh-clients libedit \
+        openssh openssh-clients libedit hwdata dbus-glib fipscheck* *devel sysvinit-tools \
         && \
+    rpm -qa groff | xargs -r rpm -e --nodeps && \
     yum clean all && \
-    bash -c 'find /usr/share/locale -maxdepth 1 -mindepth 1 -type d | grep -v -e "en_US" | xargs rm -rfv' && \
+    rpm -vv --rebuilddb && rm -rf /var/lib/yum/yumdb && rm -rf /var/lib/yum/history && \
+    find /usr/share/locale -mindepth 1 -maxdepth 1 ! -name 'en_US' ! -name 'locale.alias' | xargs -r rm -r && rm -f /var/log/*log && \
     bash -c 'localedef --list-archive | grep -v -e "en_US" | xargs localedef --delete-from-archive' && \
-    mv -f /usr/lib/locale/locale-archive /usr/lib/locale/locale-archive.tmpl && \
```

Comment:
As discussed at skype, it would be good to add some comment above this line to explain a little bit. Something like the following suggestion:

"# We use cp instead of mv due to [reason]. We use /bin/cp instead of cp to avoid alias, we in some cases has been problematic"